lib/simulator.js

```diff
@@ -92,6 +92,15 @@ const devicePairCompatiblity = exports.devicePairCompatiblity = {
 		'9.x': {        // iOS 9.0
 			'2.x': true // watchOS 2.0
 		}
+	},
+	'8.x': {
```

Comment:
Xcode typically includes the simulators for the current iOS sdk version and the previous version sims can be downloaded. If Xcode 9 come out in the future, it probably supports iOS 11 and iOS 10 sims. So we would have to edit this mapping anyway.